From d676de4db0e28c300c5eada716cb849dd636a511 Mon Sep 17 00:00:00 2001
From: Dave Eddy <dave@daveeddy.com>
Date: Thu, 24 May 2018 13:32:38 -0400
Subject: [PATCH] OS-6975 current bhyve centos image stored in manta Reviewed
 by: Jason King <jason.king@joyent.com> Reviewed by: Josh Wilsdon
 <josh@wilsdon.ca>

---
 src/vm/runtest | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/src/vm/runtest b/src/vm/runtest
index 63157cd2..0f142f8d 100755
--- a/src/vm/runtest
+++ b/src/vm/runtest
@@ -66,6 +66,12 @@ for img in $(comm -1 -3 <(eval ${installed_images}) <(eval ${want_images})); do
             && curl -kO ${MANTA_PATH}/${img}.manifest \
             && imgadm install -f ${img}.file -m ${img}.manifest \
             && rm -f ${img}.file ${img}.manifest)
+    elif [[ ${img} == "462d1d03-8457-e134-a408-cf9ea2b9be96" ]]; then
+        MANTA_PATH="https://us-east.manta.joyent.com/mgerdts/public/bhyve/images/$img"
+        (cd /var/tmp && curl -kO "${MANTA_PATH}/manifest.json" \
+            && curl -kO "${MANTA_PATH}/disk0.zfs.gz" \
+            && imgadm install -m manifest.json -f disk0.zfs.gz \
+            && rm -f manifest.json disk0.zfs.gz)
     else
         imgadm import "${img}"
     fi
-- 
2.21.0

